src/Pim/Component/Catalog/Updater/VariantGroupUpdater.php

```diff
@@ -235,17 +231,17 @@ protected function setAxes(GroupInterface $variantGroup, array $axes)
      * @param GroupInterface $variantGroup
      * @param array          $newValues
      */
-    public function setValues(GroupInterface $variantGroup, array $newValues)
+    protected function setValues(GroupInterface $variantGroup, array $newValues)
```

Comment:
No. The fact is this method should never has been public in the first time and is not used anywhere outside this class.
But you are right, I will add it.